spark/src/test/scala/org/apache/comet/exec/CometJoinSuite.scala

```diff
@@ -391,9 +391,6 @@ class CometJoinSuite extends CometTestBase {
               "AND tbl_a._2 >= tbl_b._1")
           checkSparkAnswerAndOperator(df9)
 
-          // TODO: Enable these tests after fixing the issue:
-          // https://github.com/apache/datafusion-comet/issues/861
-          /*
```

Comment:
Not just remove the comment. This test is also ignored. We need to re-enable it too.

Reply:
indeed, thanks @viirya